### PR TITLE
Add make:migration and make:seeder scripts with stubs

### PR DESCRIPTION
**Pull request by cham11ng**
Scripts to add new migration and seed file.

* **For migration**
```
$ yarn make:migration create_posts_table
```

* **For seed**
```
$ yarn make:seeder 02_insert_posts
```

And the content of the migration and seeder file is stored in `src/stubs` directory

### CODE REVIEW

**Comment by mesaugat** [comment]
Nice. Didn't know knex allowed us to provide our own stubs.

File: src/stubs/migration.stub [hunk 1]
```diff
@@ -0,0 +1,22 @@
+/**
+ * @param  {object} knex
+ * @return {Promise}
+ */
+export function up(knex) {
+  return knex.schema.createTable('table_name', table => {
+    table.increments();
+    table
```
Comment (mesaugat) [inline src/stubs/migration.stub]: Let's only add `table.timestamps()` instead.

File: src/stubs/migration.stub [hunk 2]
```diff
@@ -0,0 +1,22 @@
+/**
+ * @param  {object} knex
+ * @return {Promise}
+ */
+export function up(knex) {
+  return knex.schema.createTable('table_name', table => {
```
Comment (mesaugat) [inline src/stubs/migration.stub]: Also, migrations are not only for creating tables. We should do a tableName check on this template like it's done here. 

https://github.com/tgriesser/knex/blob/master/src/migrate/stub/js.stub#L3-L8

File: src/stubs/migration.stub [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    table.timestamp('updated_at').notNull();
+  });
+}
+
+/**
+ * @param  {object} knex
+ * @return {Promise}
+ */
+export function down(knex) {
+  return knex.schema.dropTable('table_name');
```
Comment (mesaugat) [inline src/stubs/migration.stub]: Same here. https://github.com/tgriesser/knex/blob/master/src/migrate/stub/js.stub#L12-L14

File: src/stubs/migration.stub (hunk 2 above)
Comment (cham11ng) [reply src/stubs/migration.stub]: Hmm. I've tried this before `d` returns empty object.

**Comment by cham11ng** [comment]
And I'm not aware why CI test are all failed

**Comment by mesaugat** [comment]
It will fail. Pull requests sent from forked repositories do not have access to encrypted variables or data.

**Pull request merged**